ntbs-service/DataMigration/ImportValidator.cs

```diff
@@ -281,6 +281,11 @@ private async Task CleanContactTracingData(Notification notification, PerformCon
 
             Validator.TryValidateObject(objectToValidate, validationContext, validationsResults, true);
 
+            foreach (var validationResult in validationsResults)
+            {
+                validationResult.ErrorMessage = $"{objectToValidate.GetType().Name}: {validationResult.ErrorMessage}";
```

Comment:
The trouble with using the type name here is that these messages are user facing, so it would be better to have e.g. "M. Bovis Details" rather than "MBovisDetails". I appreciate that you've followed the acceptance criteria, but I think this would be a change worth making. It looks like these classes already have a `Display` attribute, so I think you can just use that.

Reply:
Yep I can make that change.